tendrl/node_agent/ansible_runner/ansible_module_runner.py

```diff
@@ -2,10 +2,10 @@
 from ansible import modules
 import os
 import subprocess
-from tendrl.node_agent.config import TendrlConfig
+from tendrl.common.config import TendrlConfig
 import uuid
 
-config = TendrlConfig()
+config = TendrlConfig("node_agent", "/etc/tendrl/tendrl.conf")
```

Comment:
My understanding is that it could be hardcoded name and path from consumer side (node_agent in this case) but its purely generic in common module. What else you suggest here?